.github/workflows/delete-branch-on-merge-to-master.yml

```
@@ -0,0 +1,27 @@
+name: Selective delete feature branch after merge to master
+
+# Only run the workflow when a PR that targets master is closed
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - 'master'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  
+jobs:
+  merge_job:
+    # this job will only run if the PR has been merged
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+    - name: Delete merged branch
+    # from the github actions marketplace repo here: https://github.com/pabio/delete-merged-action
+      uses: koj-co/delete-merged-action@master
+      with:
+      # do not delete master, development, or hot-fix-release branches
```

Can we standardize on "hotfix" and not write it as "hot-fix"?  Hyphenating it as "hot-fix" seems quite rare -- e.g. Broad Slack has 3118 message hit for "hotfix", and 43 for "hot-fix".  

Accepting both in the code on the line below this one seems reasonable enough, but having differing conventions for how we write it would needlessly complicate e.g. naive string search.  For example, it'd be best for folks to only need to search "hotfix" and not also "hot-fix" if trying to find references to the subject via in-page find in a document, or via GitHub search in our PRs, etc.

```suggestion
      # do not delete master, development, or hotfix-release branches
```

---

I didn't want to be overly prescriptive on the wording but if everyone agrees on "hotfix" as the single best term I am totally onboard with that. @bistline @jlchang @michelle-vel curious your opinions on "hotfix"